model-server-openapi/specifications/model-server.yaml

```diff
@@ -13,481 +13,700 @@ info:
   description: "modelix public API"
   version: "1.0.0"
 servers:
-  - url: '/'
+  - url: '/v2'
     description: model-server
+
+tags:
```

Comment:
Users showed interest in a whole new king of API that can be used to read the version history.
e.g. Who made when a change. What is the hash of the change. What are previous versions? How to a normalized linear history look? How does the history graph looks?